manifest.json

```diff
@@ -21,7 +21,7 @@
       "allowFTDISerial": false,
       "allowAMASerial": false,
       "showBattery": false,
-      "debug": ""
+      "zigbee2mqttDebugLogs": false
```

Comment:
You removed the wrong one. 🙂

`debug` should remain but `zigbee2mqttDebugLogs` should be removed.

Reply:
:see_no_evil:

Reply:
Should be fixed now